dotcom-rendering/scripts/deploy/riff-raff.yaml

```diff
@@ -3,21 +3,33 @@ regions: [eu-west-1]
 allowedStages:
   - CODE
   - PROD
+templates:
+  cloudformation:
+    type: cloud-formation
+    parameters:
+      amiEncrypted: true
+      amiTags:
+        # Keep the Node version in sync with `.nvmrc`
+        Recipe: dotcom-rendering-ARM-jammy-node-18.17.0
+        AmigoStage: PROD
 deployments:
   frontend-cfn:
-    type: cloud-formation
+    template: cloudformation
     parameters:
       templateStagePaths:
         CODE: DotcomRendering-CODE.template.json
         PROD: DotcomRendering-PROD.template.json
       cloudFormationStackByTags: false
       cloudFormationStackName: rendering
       amiParameter: AMIRendering
-      amiEncrypted: true
-      amiTags:
-        # Keep the Node version in sync with `.nvmrc`
-        Recipe: dotcom-rendering-ARM-jammy-node-18.17.0
-        AmigoStage: PROD
+  render-front:
+    template: cloudformation
+    parameters:
+      templateStagePaths:
+        CODE: DotcomRendering-front-CODE.template.json
```

Comment:
I'm not sure that Riff-Raff is going to cope with this being deployed to `CODE` only. I _think_ we might need to provide a `PROD` template here too (even if we may want to scale this down initially until it's receiving 'real' traffic).

Reply:
This has been added.